ts/App.tsx

```diff
@@ -11,10 +11,38 @@ import { MenuProvider } from "react-native-popup-menu";
 import { SafeAreaProvider } from "react-native-safe-area-context";
 import { Provider } from "react-redux";
 import { PersistGate } from "redux-persist/integration/react";
+import * as Sentry from "@sentry/react-native";
 import RootContainer from "./RootContainer";
 import { persistor, store } from "./boot/configureStoreAndPersistor";
 import { LightModalProvider } from "./components/ui/LightModal";
 import theme from "./theme";
+import { sentryDsn } from "./config";
+import { isLocalEnv } from "./utils/environment";
+
+const removeUserFromEvent = (event: any) => {
```

Comment:
```suggestion
const removeUserFromEvent = (event: unknown) => {
```

Reply:
addressed in [90a64a5](https://github.com/pagopa/io-app/pull/5756/commits/90a64a5f9d63f03bcf55b86375c344ae9474352b) with correct types